unreleased_changes/8757.md

```diff
@@ -0,0 +1,2 @@
+### Changed
+- The backend can now return the update actions of an annotation per version in a truncated format to void too large responses. This does not impact restoring versions of annotations.
```

Comment:
```suggestion
- The backend can now return the update actions of an annotation per version in a truncated format to avoid too large responses. This does not impact restoring versions of annotations.
```